test/home.http

```diff
@@ -0,0 +1,4 @@
+GET http://localhost:3000/
```

Comment:
this deserves a comment to know what this request does :)

Reply:
Fixed, now has a comment.